extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -438,7 +438,7 @@ export class AmpAnalytics extends AMP.BaseElement {
     request = this.expandTemplate_(request, trigger, event);
 
     // For consistency with amp-pixel we also expand any url replacements.
-    return urlReplacementsForDoc(this.win.document).expandAsync(request)
+    return urlReplacementsForDoc(this.getAmpDoc()).expandAsync(request)
```

Comment:
yes, it is not the js/dom event object but a struct defined by instrumentation.js.